app/util/geo-utils.coffee

```diff
@@ -0,0 +1,11 @@
+toRad = (deg) -> deg * Math.PI / 180
+toDeg = (rad) -> rad * 180 / Math.PI
+
+getBearing = (lat1,lng1,lat2,lng2) ->
```

Comment:
I think something short like:

    // Get direction of movement from two first to second point.
    // North is 0, east is 90, south 180, west 270